addons/html_builder/static/src/core/builder_overlay/builder_overlay.js

```diff
@@ -43,6 +43,8 @@ export class BuilderOverlay {
             `.e:not(.o_grid_handle), .w:not(.o_grid_handle)`
         );
         this.gridHandles = this.handlesWrapperEl.querySelectorAll(".o_grid_handle");
+        this.isRtl =
+            !!this.overlayTarget.ownerDocument.body.querySelector("#wrapwrap.o_rtl");
```

Comment:
You could use the editor config here , instead of checking the wrapwrap again no ? :wink: 
Like in the builderOverlay plugin, pass it to the constructor like `isRtl: this.config.isEditableRTL` and then do `this.isRTL = isRTL` in the constructor :+1: